makefile

```diff
@@ -152,16 +152,21 @@ $(buildDir)/.load-local-data:$(buildDir)/load-smoke-data
 smoke-test-agent-monitor:$(localClientBinary) load-smoke-data
 	./$< service deploy start-evergreen --web --binary ./$< &
 	./$< service deploy start-evergreen --monitor --binary ./$< --distro localhost &
-	./$< service deploy test-endpoints --check-build --username admin --key abb623665fdbf368a1db980dde6ee0f0 $(smokeFile) || (pkill -f $<; exit 1)
+	./$< service deploy test-endpoints --check-build --username admin --key abb623665fdbf368a1db980dde6ee0f0
```

Comment:
Your explanation makes a lot of sense, and I see why it makes sense that we don't need to test the new smoke test in the sandbox. However, because there are changes to it, I think we should test the existing smoke test in the sandbox so that if something no longer passes, we can know before the merge.

Reply:
I understand, I was able to run the tasks successfully using a patch of this PR on the smoke project in staging: https://evergreen-staging.corp.mongodb.com/version/smoke_bd8d71bf821482f166fa75b3fc10f4441fde098a

Reply:
that one tests smoke-test-container-task and smoke-test-host-task. I think it would also be a good idea to test smoke-test-endpoints and smoke-test-task.